app/views/home/index.html.haml

```diff
@@ -53,14 +53,24 @@
           %th= heading_helper FinancialTransaction, :created_on
           %th= heading_helper FinancialTransaction, :user
           %th= heading_helper FinancialTransaction, :note
-          %th= heading_helper FinancialTransaction, :amount
+          %th= heading_helper FinancialTransaction, :deposit
+          %th= heading_helper FinancialTransaction, :memebership_fee
+          %th= heading_helper FinancialTransaction, :credit
         - for ft in current_user.ordergroup.financial_transactions.limit(5).order('created_on DESC')
           %tr
             %td= format_time(ft.created_on)
             %td= h(show_user(ft.user))
             %td= h(ft.note)
             - color = ft.amount < 0 ? 'red' : 'black'
-            %td{:style => "color:#{color}; width:5em", :class => "currency"}= number_to_currency(ft.amount)
+            %td.currency{:style => "color:#{color}; width:5em"}
+              - if ft.deposit?
+                = number_to_currency(ft.amount)
+            %td.currency{:style => "color:#{color}; width:5em"}
+              - if ft.membership_fee?
+                = number_to_currency(ft.amount)
+            %td.currency{:style => "color:#{color}; width:5em"}
+              - if ft.credit?
+                = number_to_currency(ft.amount)
```

Comment:
Since we do not handle the "normal credit" via the Foodsoft we need different subaccounts.
Maybe more information how we handle money in our foodcoop:
Members give money via SEPA bank transfer to the bank account of the foodcoop. The "Zahlungsreferenz" field (usually used by companies to have a uniqued id for findig the corresponding invoice) must be filled for every transaction with a special code. This code then tells how much money of should be used for the different "subaccounts". This is required to keep the transaction count low to keep the banking fees at a minimum.
Our foodsoft then imports all information directly from our ebanking and adds the different amounts to the different subaccounts.
In our stock there is a folder with a sheet of paper for every ordergroup, which keeps track of "the real account balance". After transferring money to the bank account of the foodcoop a member has to add this transaction also to this sheet of paper (but only the amount for the "credit"-type, the "membership"-cost are handled only in the foodsoft).
When buying stuff the members have to add this their sheet of paper and calculate the new total, so that there is always the current balance of their account visible.
As you can see the amount on the sheet of paper and the amount shown in the foodsoft get out of sync. Because of that we take the latest balance of the paper and insert it into the foodsoft every few months.
The "balancing"-feature for the orders in the foodsoft is not used.

Because of that offline/online-difference and to make the membership fee more transparent to the members we need this distinction.
Sometimes we have to transfer money between the different subaccounts, so splitting this information into various plugins is not an option.

FYI: We are not the only foodcoop with the "paper in the stock"-concept. So maybe find a nice solution for that, which can be accepted in upstream.